bot/exts/previewing/preview.py

```diff
@@ -1,9 +1,11 @@
 from typing import Literal
 
 from bot.bot import Bot
+from bot.utils.executor import in_executor
 from bot.utils.images import add_background, download_image, image_to_file
 from disnake.ext import commands
-from disnake.interactions.application_command import ApplicationCommandInteraction
+from disnake.interactions.application_command import \
+    ApplicationCommandInteraction
```

Comment:
```suggestion
from disnake.interactions.application_command import ApplicationCommandInteraction
```
Shouldn't this be fine? I think the `\` makes things a bit messy.

Reply:
I blame my editor's automatic import.